pkg/cliplugins/workspace/plugin/kubeconfig.go

```diff
@@ -467,3 +468,57 @@ func (kc *KubeConfig) CreateContext(ctx context.Context, name string, overwrite
 
 	return nil
 }
+
+// enableSyncer prepares a kcp workspace for use with a syncer and outputs the
+// configuration required to deploy a syncer to the pcluster to stdout.
+func (kc *KubeConfig) EnableSyncer(ctx context.Context, workloadClusterName, image string, resourcesToSync []string, disableDeployment bool) error {
```

Comment:
not sure what this has to do with kubeconfig.go. Please move all of this to cliplugins/workload or so.